### PR TITLE
[DOCS-475] SAML Configuration Instructions

### PR DESCRIPTION
**Pull request by ana-dashuk-cobalt**
### Updated

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
| Configure SAML SSO | [Link](https://deploy-preview-357--cobalt-docs.netlify.app/platform-deep-dive/organization/organization-settings/saml-sso/) | |

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

### CODE REVIEW

File: content/en/Platform Deep Dive/Organization/Organization Settings/SAML SSO/_index.md [hunk 1]
```diff
@@ -114,55 +121,69 @@ To configure SAML SSO with Azure Active Directory (Azure AD):
...
+    - **NameID format**: `urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress`
+    - **NameID attribute**: `mail`
+    - **Signature algorithm**: `SHA256`
+    - **Signing options**: Select **Sign response** and **Sign assertion**.
+    - **Map attributes**:
+      - **IdP Attribute**: `mail`
+      - **SAML Response Attribute**: `email`
+1. Complete other required steps.
+1. Test the integration.
+1. If the test is successful, assign users to the application.
```
Comment (molfinn) [inline content/en/Platform Deep Dive/Organization/Organization Settings/SAML SSO/_index.md:155]: I think you need to have users assigned to the app before you test? Unless this test option is within Okta.

File: content/en/Platform Deep Dive/Organization/Organization Settings/SAML SSO/_index.md [hunk 2]
```diff
@@ -94,7 +94,10 @@ Learn how to configure SAML SSO with Azure Active Directory (Azure AD) as IdP.
 <br>
 To configure SAML SSO with Azure Active Directory (Azure AD):
 
-1. In Azure AD, create an [enterprise non-gallery application](https://learn.microsoft.com/en-us/azure/active-directory/manage-apps/add-application-portal) for Cobalt.
+1. In Azure AD, create an enterprise non-gallery application for Cobalt.
+    - Select **Enterprise applications**, then select **New application**.
```
Comment (molfinn) [inline content/en/Platform Deep Dive/Organization/Organization Settings/SAML SSO/_index.md:98]: just showing you the UI. I want to make sure folks don't miss the "Create your own application" https://www.loom.com/share/3a0744b8f7894e908b0c8678a49e26f1

Comment (ana-dashuk-cobalt) [reply content/en/Platform Deep Dive/Organization/Organization Settings/SAML SSO/_index.md:98]: Thank you, updated. I have also removed other links to Microsoft instructions.

File: content/en/Platform Deep Dive/Organization/Organization Settings/SAML SSO/_index.md [hunk 3]
```diff
@@ -173,55 +197,74 @@ Once you've completed the setup, your application for Cobalt appears in the Goog
 
 If you create an application for Okta **manually**, use the following parameters.-->
 
-We recommend creating a non-gallery SAML application for Cobalt manually. For details, read Okta's [documentation](https://help.okta.com/en-us/Content/Topics/Apps/Apps_App_Integration_Wizard_SAML.htm?cshid=ext_Apps_App_Integration_Wizard-saml).
-
-| For This Parameter in Cobalt | Enter This Value from Okta |
-|---|---|
-| IdP SSO URL | Sign on URL (Identity Provider Single Sign-On URL) |
-| IdP Certificate | Signing Certificate (X.509 Certificate) |
+We recommend creating a **non-gallery SAML application** for Cobalt manually. For more information, refer to Okta [documentation](https://help.okta.com/en-us/Content/Topics/Apps/Apps_App_Integration_Wizard_SAML.htm?cshid=ext_Apps_App_Integration_Wizard-saml).
```
Comment (molfinn) [inline content/en/Platform Deep Dive/Organization/Organization Settings/SAML SSO/_index.md:200]: I think we should not recommend, but tell them. The Cobalt gallery in Okta will NOT work with Auth0 until it's updated, which won't be for some time.

**Review by molfinn — APPROVED** [review]

**Pull request merged**